website/content/docs/connect/config-entries/sameness-group.mdx

```diff
@@ -147,9 +146,23 @@ Specifies the local admin partition that the sameness group applies to. Refer to
 
 ### `DefaultForFailover`
 
-Determines whether the sameness group should be used to establish connections to services with the same name during failover scenarios. When this field is set to `true`, DNS queries and upstream requests automatically failover to services in the sameness group according to the order of the members in the `Members` list.
+Determines whether the sameness group should be used to establish connections to services with the same name during failover scenarios.
+
+When this field is set to `true`, DNS queries and upstream requests automatically failover to services in the sameness group according to the order of the members in the `Members` list. It impacts all services on the partition.
+
+When this field is set to `false`:
+  
+  - You must specify `destination_sameness_group` in order to configure an upstream. Refer to [Upstream configuration reference](/consul/docs/connect/proxies/proxy-config-reference#upstream-configuration-reference) for more information. 
+  - You can use a sameness group for failover by configuring the `Failover` block of a [service resolver configuration entry](/consul/docs/connect/config-entries/service-resolver).
+
+#### Values
+
+- Default: `false`
+- Data type: Boolean
+
+### `IncludeLocal`
 
-When this field is set to `false`, you can still use a sameness group for failover by configuring the `Failover` block of a [service resolver configuration entry](/consul/docs/connect/config-entries/service-resolver).
+Determines whether the local partition should be considered the first member of the sameness group. When this field is set to `true`, DNS queries, upstream requests, and failover traffic target return a health instance from the local partition unless one does not exist.
```

Comment:
Right below this section is the `Members` field -- all the `IncludeLocal` does is simply implicitly add the local partition as the first item in that `Members` listing.

Also worth noting is this description in the `Members` section.

>Specifies a partition in the local datacenter that is a member of the sameness group. When the value of this field is set to `*`, all local partitions become members of the sameness group.

I believe this is wrong. I don't think we support wildcard configuration for sameness groups.